app/src/main/java/com/blackboxteam/app/blackbox/util/Validation.java

```diff
@@ -0,0 +1,35 @@
+package com.blackboxteam.app.blackbox.util;
+
+import android.text.TextUtils;
+import android.util.Patterns;
+
+/**
+ * Created by andeladeveloper on 07/06/2017.
+ */
+
+public class Validation {
+
+    public static boolean validateIfEmpty(String name) {
+        if (TextUtils.isEmpty(name)) {
```

Comment:
The body of this function could be a one-liner. `return !TextUtils.isEmpty(name)`